strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -679,6 +685,37 @@ public void testChecksumsStorage(@MavenRepository(repositoryId = "checksums-stor
         assertThat(actualChecksums).isEqualTo(expectedChecksums);
     }
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
+    @Test
+    public void testFailArtifactStorageWhenChecksumIsBroken(@MavenRepository(repositoryId = "broken-checksum-test")
+                                                            @RepositoryAttributes(strictChecksum = true)
+                                                            Repository repository,
+                                                            @MavenTestArtifact(repositoryId = "broken-checksum-test",
+                                                                               id = "org.carlspring.strongbox:strongbox-test-artifact",
+                                                                               versions = { "1.0" })
+                                                            Path artifactPath)
+            throws IOException
+    {
+        final String storageId = repository.getStorage().getId();
+        final String repositoryId = repository.getId();
+        final String checksumFilePath = "org/carlspring/strongbox/strongbox-test-artifact/1.0/strongbox-test-artifact-1.0.jar.md5";
+
+        RepositoryPath repositoryPath = repositoryPathResolver.resolve(storageId,
+                                                                       repositoryId,
+                                                                       checksumFilePath);
+        Files.write(repositoryPath, "broken_checksum".getBytes());
+
+        try (InputStream is = Files.newInputStream(repositoryPath))
+        {
+            Throwable throwable = catchThrowable(() -> mavenArtifactManagementService.store(repositoryPath,is));
+
+            final String fullChecksumFilePath = String.format("/%s/%s/%s", storageId, repositoryId, checksumFilePath);
+            assertThat(throwable)
+                    .isExactlyInstanceOf(ArtifactStorageException.class)
+                    .hasMessage("Invalid checksum [broken_checksum] for artifact [strongbox:%s]", fullChecksumFilePath);
```

Comment:
Cosmetic: Would you mind re-aligning this code like this:
```
            assertThat(throwable).isExactlyInstanceOf(ArtifactStorageException.class)
                                 .hasMessage("Invalid checksum [broken_checksum] for artifact [strongbox:%s]",
                                             fullChecksumFilePath);
```

Reply:
Done!